vrk-kpa/suomidigi
# SUOPA-860 Legislation prev/next buttons

To test: 
- Open proto: https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/128cc938-350b-482c-a8da-8d28e800a53d/Suomidigi-2020-lainsaadanto-proto1-2-XL-1
- Log in and go to https://suomidigi.docker.sh/ohjeet-ja-tuki/lainsaadanto/digipalvelulaki
- Scroll down until you see "4 § Digitaalisten palvelujen suunnittelu ja ylläpito"
- Click on the first link on the list. You should end up on a legislation card: https://suomidigi.docker.sh/node/139
   - Edit the node and select "Valikon asetukset" -> "Yläkohta: Laki digitaalisten palvelujen..."
   - Select "Osoitealias" -> check "Luo automaattinen osoitealias"
   - Save the node
- Check that Next and Previous links are similar to the ones in the proto. 
   - Click on next, it should point to https://suomidigi.docker.sh/node/140
   - On node 140, click on previous, it should point back to node 139
   - On node 139, click on previous and you should end up on the Digipalvelulaki - Legislation collection page you started from. 
- Finally go to the last node in the list https://suomidigi.docker.sh/node/147
   - Check that there is no Next link
